src/lib/sitemap/integrations-content-urls.ts

```diff
@@ -0,0 +1,69 @@
+import { fetchAllIntegrations } from 'lib/integrations'
+import {
+	Integration,
+	IntegrationComponent,
+} from 'lib/integrations-api-client/integration'
+import { makeSitemapElement } from './helpers'
+import { SitemapElement } from './types'
+
+// product.slug/integrations/organization.slug/slug
+export async function allIntegrationsUrls() {
+	const allPaths = await fetchAllIntegrations([
+		'vault',
+		'waypoint',
+		// vault & waypoint are currently the only products with integrations pages,
+	])
```

Comment:
This array can be replaced with an array of products constant once DevDot Integrations projects are complete

Reply:
💭 I think you should be able to use `__config.dev_dev.product_slugs_with_integrations` for this array